addons/web/models/base_document_layout.py

```diff
@@ -278,3 +281,6 @@ def _compile_scss(self, scss_source):
             )
         except libsass.CompileError as e:
             raise libsass.CompileError(e.args[0])
+
+    def _get_layout_background(self):
+        return base64.b64encode(open(os.path.join(tools.config['root_path'], 'addons', 'base', 'static', 'img', 'bg_background_template.jpg'), 'rb').read()).decode()
```

Comment:
Unclosed file descriptor + bad usage of open (should use `odoo.tools.file_open`)